src/instruments/src/EFB/Components/Form/SimpleInput/SimpleInput.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+type SimpleInputProps = {
+    label?: string,
+    placeholder?: string,
+    value?: any,
+    onChange?: (value: string) => void,
+    min?: number,
+    max?: number,
+    number?: boolean,
+    noLeftMargin?: boolean,
+    reverse?: boolean // Flip label/input order,
+    className?: string
+};
+
+type SimpleInputState = {
+    value: string
+};
```

Comment:
Not resolved

Reply:
Will be resolved by the original creator of the Landing performance PR